packages/patternfly-4/react-core/src/components/Tabs/Tabs.js

```diff
@@ -45,22 +43,33 @@ const defaultProps = {
   id: null,
   isFilled: false,
   isSecondary: false,
-  leftScrollAriaLabel: 'Scroll left',
-  rightScrollAriaLabel: 'Scroll Right',
-  'aria-label': null,
+  'aria-label-left-scroll': 'Scroll left',
+  'aria-label-right-scroll': 'Scroll Right',
   variant: TabsVariant.div
 };
 
-const Tabs = ({ 'aria-label': ariaLabel, id, variant, ...props }) => {
+const Tabs = ({ id, variant, ...props }) => {
   const unique_id = id ? id : getUniqueId();
   return variant === TabsVariant.nav
-    ? <TabsWithNav ariaLabel={ariaLabel} id={unique_id} {...props} />
+    ? <TabsWithNav id={unique_id} {...props} />
```

Comment:
Currently there's an example labeled **Accessible primary tabs with aria-label** that shows the `<nav>` variant with `aria-label`. I'm not seeing this example in the preview.

Reply:
See previous comment.